integreat_cms/cms/views/events/event_form_view.py

```diff
@@ -236,6 +236,13 @@ def post(self, request, **kwargs):
                     language__in=languages
                 ).update(status=status.DRAFT)
 
+            if (
```

Comment:
```suggestion
            elif (
```
We don't need to check this condition when the status is `DRAFT`